src/app/components/MessageBanner/index.tsx

```diff
@@ -0,0 +1,82 @@
+/** @jsx jsx */
+/* @jsxFrag React.Fragment */
+import React, { useContext } from 'react';
+import { jsx } from '@emotion/react';
+import Paragraph from '../Paragraph';
+import Heading from '../Heading';
+import Image from '../Image';
+import Text from '../Text';
+import { LeftChevron, RightChevron } from '../icons';
+import styles from './index.styles';
+import { ServiceContext } from '../../contexts/ServiceContext';
+
+interface MessageBanner {
+  summaries: {
+    imageUrl?: string;
+    link: string;
+    imageAlt?: string;
+    description?: string;
+    title: string;
+    id: string;
+  }[];
+  title: string;
+  position: number;
+}
+const MessageBanner = ({ summaries, title, position = 0 }: MessageBanner) => {
```

Comment:
Note: we are passing in position here in order to ensure that the id of the heading is unique in the case of having multiple message banners on a page. This `position` param makes more sense in the context of where it's called from - see https://github.com/bbc/simorgh/pull/10487/files#diff-92057634d5283330a6167da6a5a33b6e5d75b3fed7e09cc1de94f12e797702abR117